mappings/net/minecraft/item/BundleItem.mapping

```diff
@@ -12,5 +16,12 @@ CLASS net/minecraft/class_5537 net/minecraft/item/BundleItem
 		ARG 0 stack
 	METHOD method_31563 getBundleOccupancy (Lnet/minecraft/class_1799;)I
 		ARG 0 stack
+	METHOD method_31564 (Lnet/minecraft/class_1799;)I
+		ARG 0 heldStack
```

Comment:
What is `heldStack`?

Reply:
It's an item stack that is held in a bundle. Maybe `contentStack` is better?